pageserver/src/consumption_metrics.rs

```diff
@@ -300,35 +200,29 @@ pub async fn collect_metrics_task(
 
     // Send metrics.
     // Split into chunks of 1000 metrics to avoid exceeding the max request size
-    const CHUNK_SIZE: usize = 1000;
     let chunks = current_metrics.chunks(CHUNK_SIZE);
 
-    let mut chunk_to_send: Vec<ConsumptionMetric> = Vec::with_capacity(1000);
+    let mut chunk_to_send: Vec<Event<Ids>> = Vec::with_capacity(1000);
```

Comment:
`1000` magic constant. Is this `CHUNK_SIZE`?

Also, should update the `1000` in the comment above.

Reply:
thank you! fixed

Reply:
Comment  is still missing:

```
// Split into chunks of 1000 metrics to avoid exceeding the max request size
```